enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Visualization.tsx

```diff
@@ -44,11 +43,12 @@ export const QuestionVisualization = ({
   }
 
   // `isCardError: true` when the entity ID couldn't be resolved
```

Comment:
nit: I think we can remove this comment as we don't check for `isCardIdError` anymore